main/index.ts

```diff
@@ -63,7 +79,7 @@ log.info(`Node: v${process.versions.node}`)
 // prevent showing the exit dialog more than once
 let closing = false
 
-process.on('uncaughtException', e => {
+process.on('uncaughtException', (e) => {
```

Comment:
no need for parentheses here

Reply:
Personally I think parens are *always* better with fat arrows.  More visually distinct, especially when using a lot of them, and I believe the fewer keystrokes / less cognitive overhead required when adding a new parameter more than makes up for the occasional unnecessary characters.